cas-server-support-x509/src/test/resources/ldif/users-x509.ldif

```diff
@@ -0,0 +1,38 @@
+#
+# Licensed to Apereo under one or more contributor license
+# agreements. See the NOTICE file distributed with this work
+# for additional information regarding copyright ownership.
+# Apereo licenses this file to you under the Apache License,
+# Version 2.0 (the "License"); you may not use this file
+# except in compliance with the License.  You may obtain a
+# copy of the License at the following location:
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+#
+
+dn: CN=x509,ou=people,dc=example,dc=org
+objectClass: top
+objectClass: person
+objectClass: organizationalPerson
+objectClass: ad
+objectClass: inetOrgPerson
+objectClass: cRLDistributionPoint
+cn: Misagh Moayyed
+sn: Moayyed
+givenName: Misagh
+displayName: Misagh Moayyed
+sAMAccountName: mmoayyed
+mail: mmoayyed@example.org
+userPrincipalName: mmoayyed@example.org
+userAccountControl: 66048
+distinguishedName: CN=mmoayyed,ou=people,dc=example,dc=org
+userPassword: misagh
+unicodePwd: ignoredForTests
+certificateRevocationList;binary:< file:cas-server-support-x509/src/test/resources/userCA-valid.crl
```

Comment:
It's not, yes, because it already exists as an available resource in the x509 module.